### PR TITLE
Refresh the documentation about deploying a registry

### PR DESCRIPTION
**Pull request by dcermak**
### PR creator: Description

Refresh the section about deploying a registry to use the SLE BCI Registry image.

cc @danishprakash @dirkmueller

### CODE REVIEW

**Review by dmpop — APPROVED** [review]
Looks good to me! Thank you, Dan!

File: container-guide/concepts/containers-docker-registry.xml [hunk 1]
```diff
@@ -49,36 +49,33 @@
...
-      addr: 0.0.0.0:5000</screen>
-    <para>
-      Also create an empty directory to map the
-      <filename>/var/lib/docker-registry</filename> directory outside the
-      container. This directory is used for storing container images.
+      run a local &dreg; as a container. It stores the pushed images in a
+      container volume corresponding to the directory
+      <filename>/var/lib/docker-registry</filename>. It is recommended to
+      either create a named volume for the registry or to bind mount the
+      directory <filename>/var/lib/docker-registry</filename> onto the host, to
```
Comment (danishprakash) [inline container-guide/concepts/containers-docker-registry.xml:56]: > onto the host

"into the container"? it reads like the bind mount is operating in an inverted manner

Comment (dcermak) [reply container-guide/concepts/containers-docker-registry.xml]: I've tweaked the sentence a bit, wdyt?

Comment (danishprakash) [reply container-guide/concepts/containers-docker-registry.xml]: Perfect, thanks!

**Pull request merged**